### ISSUE TITLE
Additional functionality for IP Set manipulation

### ISSUE THREAD

**Issue by dcoghlan** [vmware/powernsx#161]
I am working on a customer site at the moment ;) and I would like the ability to be able to set/update a given IP Set object to add or remove values from the configured values.

something along the following lines is essentially what I would like to see.

```
get-nsxipset -name ND2Address | set-nsxipset -add -IPAddress 1.1.1.1
get-nsxipset -name ND2Address | set-nsxipset -add -Range 1.1.1.1-1.1.1.254
get-nsxipset -name ND2Address | set-nsxipset -add -Prefix 1.1.1.1/24
get-nsxipset -name ND2Address | set-nsxipset -remove -IPAddress 1.1.1.1
get-nsxipset -name ND2Address | set-nsxipset -remove -Range 1.1.1.1-1.1.1.254
get-nsxipset -name ND2Address | set-nsxipset -remove -Prefix 1.1.1.1/24
```

**Comment by nmbradford** [comment]
Understood.  About to embark on a long plane flight, so this sounds doable! :).  Ill aim for add-nsxipsetmember and remove-nsxipsetmember, but the functionality seems obvious.  Thanks @dcoghlan

**Comment by nmbradford** [comment]
resolved in #165 

update and enjoy!
